packages/core/src/knapsack-pro-logger.ts

```diff
@@ -1,9 +1,8 @@
 import { inspect } from 'util';
 import { createLogger, format, Logger, transports } from 'winston';
-import { KnapsackProEnvConfig } from './config';
+import { KnapsackProEnvConfig } from './config/index.js';
 
-// eslint-disable-next-line @typescript-eslint/no-require-imports
-const { name: clientName } = require('../package.json');
+const clientName = '@knapsack-pro/core';
```

Comment:
suggestion 

Perhaps a better place to keep the definition of the name is somewhere in src/config like src/config/index.js or src/config/consts.js